content/faq/faq.mdx

```diff
@@ -145,6 +147,107 @@ Now default values within the schema file can be used:
   "default": "Max"
 }
 ```
+
+## How to update a field based on another field in JSON Forms?
+
+Consider a carwash application offering various services and calculating the resulting price.
+The schema includes a multiselect field called services and a price attribute, with the price field set as readonly.
+
+<DependentFieldExample/>
+
+There are two approaches to update a field in JSON Forms based on the value of another field:
+
+#### Approach 1: Implementing functionality in the onChange method of JSON Forms
+
+In this approach, you can implement the logic in the onChange method of JSON Forms. 
+The onChange method triggers whenever a user changes the form data.
+
+```js
+export const CarWash = () => {
+  const [formData, setFormData] = useState(inputData);
+  
+  const onChange = ({ data, _errors }) => {
+    const price = data.services.length*15;
+    if (data.price === price){
+        setFormData(data);
+    }else{
+        setFormData({...data, price: price})
+    }
+  }
+
+  return (
+    <JsonForms
+      data={formData}
+      schema={schema}
+      renderers={renderers}
+      onChange={onChange}
+    />
+  );
+};
+```
+
+In the onChange method, the price is calculated based on the selected services. 
+If the calculated price matches the existing value, the data remains unchanged.
+However, if the price differs, a new data object is created with the updated price and passed to JSON Forms.
+It's important to only create a new object when necessary to avoid infinite loops.
```

Comment:
This sentence should be in a warning element. We already have some in the website.